packages/core/src/interfaces/telementry.ts

```diff
@@ -0,0 +1,14 @@
+export type ITelemetryData = {
+    providers: {
+        authProvider?: boolean;
+        dataProvider?: boolean;
+        routerProvider?: boolean;
+        notificationProvider?: boolean;
+        liveProvider?: boolean;
+        auditLogProvider?: boolean;
+        i18nProvider?: boolean;
+        accessControlProvider?: boolean;
```

Comment:
```suggestion
        auth?: boolean;
        data?: boolean;
        router?: boolean;
        notification?: boolean;
        live?: boolean;
        auditLog?: boolean;
        i18n?: boolean;
        accessControl?: boolean;
```
could we remove "Provider" suffix?

Reply:
Encryption `object` does not support. We can do something like: 

```
  usesAuth?: boolean;
  usesAL?: boolean;
  usesLive?: boolean;
  usesRouter?: boolean;
  usesData?: boolean;
  usesI18n?: boolean;
  usesNotification?: boolean;
  usesAC?: boolean;
  version: string;
  resourceCount: number;
```